buildone

```diff
@@ -43,8 +43,9 @@ log() {
 
 build() {
     DIST=$1
+    PLATFORM=${2:-amd64}
```

Comment:
I believe we should consider the $PLATFORM name when querying the registry in:

```
if docker pull "$BASENAME:$TAG" > /dev/null; then
   ...
}
```

If not, when comparing the amd64 image (at least now that we only have a amd64 one, but is not labeled -amd64), the check at then end is probably going to determine the image is up to date:

```
  else
            log "Image didn't change"
            return
        fi
```

And so it will skip tagging it:

```
docker tag "$built_image_id" "$BASENAME:$TAG-$PLATFORM"
```

This may not be a problem when all is up and running, but at least for the first time, it will fail because the tag does not exists when creating the manifest.

Maybe just appending it at the beginning would work:

```
if [ -n "$debian_snapshot_id" ]; then
        TAG="${DIST}-snapshot-${debian_snapshot_id}-$PLATFORM"
    else
        TAG=$DIST-$PLATFORM
    fi
```

That would also require tweaking the "test" script to check the image name correspond to jessie. From:

`if [ "jessie" == "$DIST" ]; then`

to something like

`if [[  "$DIST" == "jessie"* ]]; then`

Reply:
Fixed